bedrock/products/views.py

```diff
@@ -36,6 +36,18 @@ def vpn_landing_page(request):
     return l10n_utils.render(request, template_name, context)
 
 
+@require_safe
+def vpn_desktop_page(request):
+    template_name = 'products/vpn/desktop/index.html'
+
+    context = {
+        'allowed_countries': vpn_allowed_countries(),
+        'default_monthly_price': vpn_default_monthly_price(),
```

Comment:
I'm not sure what the plan is for these SEO pages, but the way we're doing pricing for VPN is going to change in the coming weeks and will likely need a refactor. This is just a flag that if you use this here it will need changing later on.

Reply:
Right now we do not have any pricing on the pages, so we should wait how the landing page does it.

Reply:
This hads been removed!